internal/exportTypescriptSchema.ts

```diff
@@ -0,0 +1,28 @@
+import { generateTypeScript } from "./generateTypeScript";
+import { foxgloveEnumSchemas, foxgloveMessageSchemas } from "./schemas";
+
+/**
+ * Export typescript schema as source, keyed by the schema name.
+ *
+ * @returns a record of schema name => schema source.
+ */
+export function exportTypescriptSchema(): Record<string, string> {
+  const schemas: Record<string, string> = {};
+
+  for (const schema of Object.values(foxgloveMessageSchemas)) {
+    schemas[schema.name] = generateTypeScript(schema);
+  }
+  for (const schema of Object.values(foxgloveEnumSchemas)) {
+    schemas[schema.name] = generateTypeScript(schema);
+  }
+  const allSchemaNames = [
+    ...Object.values(foxgloveMessageSchemas),
+    ...Object.values(foxgloveEnumSchemas),
+  ].sort((a, b) => a.name.localeCompare(b.name));
+  let indexTS = "";
+  for (const schema of allSchemaNames) {
+    indexTS += `export * from "./${schema.name}";\n`;
+  }
+  schemas["index"] = indexTS;
```

Comment:
I think this also needs to include `TIME_TS` and `DURATION_TS`.

Could we also use this new function [here](https://github.com/foxglove/schemas/blob/main/scripts/updateGeneratedFiles.ts#L106) to reduce duplication?